openwisp_radius/api/views.py

```diff
@@ -14,3 +55,24 @@ def _create_batch(self, serializer, **kwargs):
 
 
 batch = BatchView.as_view()
+
```

Comment:
why are we not adding it to `BatchView`?

Let's try to avoid repeating the same statmements, try something like:

```python
class TokenAuthorizationMixin(object):
    authentication_classes = (TokenAuthentication,)


class AuthorizeView(TokenAuthorizationMixin, BaseAuthorizeView):
    pass

# etc.
```

Reply:
Some tests are failing if I add the ``Mixin`` to ``BatchView``. It seems that in django-freeradius, the View is without any ``authentication_classes`` https://github.com/openwisp/django-freeradius/blob/master/django_freeradius/api/views.py#L170

Reply:
Damn.. I'm going to fix that